config/config.exs

```diff
@@ -1,3 +1,3 @@
 use Mix.Config
 
-config :clickhousex, codec: Clickhousex.Codec.RowBinary
+# config :clickhousex, codec: Clickhousex.Codec.RowBinary
```

Comment:
I didn't see `plausible/analytics` setting the codec, so to avoid problems with it, I'm removing this setting as in [the fork](https://github.com/plausible/clickhousex/blob/e6b7cc5a1ea7bb26d77af8fdae03952f662469af/config/config.exs#L3).